site/src/api/api.ts

```diff
@@ -4,6 +4,41 @@ import * as Types from "./types"
 import { WorkspaceBuildTransition } from "./types"
 import * as TypesGen from "./typesGenerated"
 
+export const hardCodedCSRFCookie = (): string => {
+  // This is a hard coded CSRF token/cookie pair for local development.
+  // In prod, the GoLang webserver generates a random cookie with a new token for
+  // each document request. For local development, we don't use the Go webserver for static files,
+  // so this is the 'hack' to make local development work with remote apis.
+  // The CSRF cookie for this token is "JXm9hOUdZctWt0ZZGAy9xiS/gxMKYOThdxjjMnMUyn4="
+  const csrfToken =
+    "KNKvagCBEHZK7ihe2t7fj6VeJ0UyTDco1yVUJE8N06oNqxLu5Zx1vRxZbgfC0mJJgeGkVjgs08mgPbcWPBkZ1A=="
+  axios.defaults.headers.common["X-CSRF-TOKEN"] = csrfToken
+  return csrfToken
+}
```

Comment:
great docs on this 👍

Reply:
> For local development, we don't use the Go webserver for static files

@Emyrk Dumb question...why don't we use the Go webserver for static files? What do we instead? (Just trying to understand the whole picture).

Reply:
We do, but for the FE team to have a hot reload, we have to use webpack server instead to host the files. So the webpack server has to do what the golang server is doing.

Instead of implementing CSRF completely, we just hardcode it since it's only used for developement.